docs/guide-to-triaging.md

```diff
@@ -0,0 +1,158 @@
+
+- Sniffs data - the new hotness. 'Sniffs' are intercepted data packets players have captured when playing retail Wow. They have the advantage of using real retail data and making it available in SQL, and thus broadly useable. The disadvantages are that the sniff data only contains client-visible data, it is often incomplete and patchy, and is often difficult to otherwise verify. 
+    For one useful site that makes use of this data, see the [AzerothCore SpeedChecker](https://azerothcore-speedchecker.web.app/) by @Efymer on Discord - if you want to verify a creature's movement speed or other flags versus sniffed retail data, this will do it for you.
+
+- [Wayback Machine/Internet Archive](https://web.archive.org/) - if you really get stuck for contemporary sources and need to see the state of a site in 2010, then Wayback Machine can be good, if you know the URL of the site you want to see. For example, they have a 2010 snapshot of the hunter pet site Petopia which has occasionally been useful.
+
+- Other forums (Wow/MMO-Champion) - the Wow forums can occasionally be useful when they discuss vanilla or TBC issues, but are usually full of people contradicting and flaming each other, so aren't very reliable as sources.
+
+- https://www.mmo-champion.com/forum.php - Similarly, MMO-Champion has forums that can be useful because they have been going, essentially unchanged, since before Wrath. This means you can sometimes find useful firsthand information there. However, like most forums, sorting the useful information from the random chaff can be difficult.
+
+- https://wowgaming.altervista.org/aowow/ - this is a site acting as the front end to a snapshot of our own AC database. As a result, it provides a rapid way of seeing
+
+- https://wotlkdb.com/ - another third-party snapshot of Wrath-era data, although the quality of its data is occasionally questionable. Not a primary source.
+
+- The triaging Discord's #need-help channel is very useful here.
+
+## Tools
+- The AC server/test environment, This is the basic testing tool. Try to keep your AC server as up-to-date as you can manage. Daily updates are a good idea, and you shouldn't go more than a week without updating. 
+
+- GM commands and macros - see https://www.azerothcore.org/wiki/gm-commands At a bare minimum you want to be familiar with the various cheats (no damage, speed, fly), `.go`, `.additem`, `.quest add` and so on. It can also be useful to setup various cheat and boost macros on your test account, so you can instantly setup and train a new character. When testing various class and racial issues you find yourself making and then deleting a lot of different characters, so automating the new character setup with macros can repay the time invested quite quickly. 
```

Comment:
Add a link, dont paste the full URL. Also use relative links when linking to a wiki page. See https://azerothcore.org/wiki/wiki-standards

Reply:
Fixed.